glossary.md

```diff
@@ -101,16 +168,32 @@ Currently, available steps generated by PD include:
 - `PromoteLearner`: Promotes a specified learner to a voting member
 - `SplitRegion`: Splits a specified Region into two
 
+### Optimistic transaction
+
+Optimistic transactions are transactions that use optimistic concurrency control and generally do not cause conflicts in concurrent environments. After enabling optimistic transactions, TiDB only checks for conflicts when the transaction is finally committed. The optimistic transaction mode is suitable for concurrent scenarios with more reads and fewer writes, which can improve the performance of TiDB.
+
+Starting from v3.0.8, TiDB clusters default to pessimistic transaction mode. However, if a cluster created before v3.0.7 is upgraded to v3.0.8 or later, the default transaction mode will not be changed; only newly created clusters will default to pessimistic transaction mode. For details, refer to [TiDB Optimistic Transaction Model](/optimistic-transaction.md).
```

Comment:
```suggestion
```

I don't think we need to have detail like this in the docs for current versions that goes back to v4.x or earlier.